newswires/app/db/FingerpostWireEntry.scala

```diff
@@ -35,7 +35,8 @@ object FingerpostWireEntry extends SQLSyntaxSupport[FingerpostWireEntry] {
   implicit val format: OFormat[FingerpostWireEntry] =
     Json.format[FingerpostWireEntry]
 
-  override val columns = Seq("id", "external_id", "ingested_at", "content")
+  override val columns =
```

Comment:
this doesn't have the side-effect that the generated column is returned from the DB for each row does it?

Reply:
ah good point! not this line, but the fact that there's a new column in the table will affect any `select *`s yep. we should update them to only select the columns we actually want...

Reply:
fixed